templates/secrets_datastore_crypto_key.yaml

```
@@ -2,12 +2,13 @@
 {{- $deprecated_crypto_key := (default (dict) (default (dict) .Values.secrets).st2).datastore_crypto_key }}
 {{- if $deprecated_crypto_key }}
 {{- fail "Please update your values! The datastore_crypto_key value moved from secrets.st2.* to st2.*" }}
-{{- else if .Values.st2.datastore_crypto_key }}
+{{- end }}
```

If we need to support reducing security by disabling the datastore_crypto_key, then we could do something like:
```suggestion
{{- else if .Values.st2.datastore_crypto_key != "disable" }}
```
And do something similar everywhere else I've remoed this check.